### PR TITLE
[DOCU-2165] add Send First Request page

### PR DESCRIPTION
**Pull request by falondarville**
### Summary

Add send first request instructions. The audience is people who have not used an API client before, or people who are using Insomnia for the very first time with minimal experience of an API client. 

I have also linked this new doc from the Insomnia Docs homepage. 

### Reason

Resolves https://linear.app/insomnia/issue/INS-1285/sending-request-docs

New doc: https://insomnia-docs-git-docu-2165-teaminsomnia.vercel.app/insomnia/send-your-first-request
Link from homepage: https://insomnia-docs-git-docu-2165-teaminsomnia.vercel.app/

### CODE REVIEW

File: docs/insomnia/send-your-first-request.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+1. From your Insomnia dashboard, create a new Document or Collection.
+1. In a Document, select the **Debug** tab, then **New Request**. In a Collection, select **New Request**.
+1. In the **New Request** modal:
+   1. Enter a request name like `Return mockbin message`.
+   1. Select a `GET` method from the dropdown.
+   1. Click **Create**.
+1. Enter your mockbin URL in the request URL input.
+1. Click **Send**.
+
+   ![Your mockbin messsage should appear in the Preview panel.](/assets/images/mockbin-return.png)
```
Comment (dimitropoulos) [inline docs/insomnia/send-your-first-request.md:55]:
```suggestion
   ![Your mockbin message should appear in the Preview panel.](/assets/images/mockbin-return.png)
```

File: docs/insomnia/send-your-first-request.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+---
+layout: article-detail
+title:  Send Your First Request
+category: "Get Started"
+category-url: get-started
+---
+
+Insomnia is an API client that enables you to send API requests outside of your terminal, and without writing any code. This means you don't have to write any finnicky cURL requests or backend code to quickly retrieve information, push content to a database, and do anything else a given API enables you to do.
```
Comment (dimitropoulos) [inline docs/insomnia/send-your-first-request.md:8]:
```suggestion
Insomnia is an API client that enables you to send API requests outside of your terminal, and without writing any code. This means you don't have to write any finicky cURL requests or backend code to quickly retrieve information, push content to a database, and do anything else a given API enables you to do.
```

**Review by dimitropoulos — APPROVED** [review]
two tiny typos: but otherwise looks good!  I'm really happy to see us writing little codelabs like this!

**Comment by falondarville** [comment]
Thank you so much @dimitropoulos ! Excited to have this now!

**Pull request merged**